uuid_test.go

```diff
@@ -569,7 +569,7 @@ func TestWrongLength(t *testing.T) {
 func TestIsWrongLength(t *testing.T) {
 	_, err := Parse("12345")
 	if !IsInvalidLengthError(err) {
-		t.Errorf("expected error type is invalidLengthError")
+		t.Errorf("expected error type is ErrInvalidLength")
```

Comment:
Technically this is incorrect at `ErrInvalidLength` is an error and not a type.  This should be something like `"IsInvalidLength returned incorrect type %T", err`

Reply:
Solved in [54f9572](https://github.com/google/uuid/pull/166/commits/54f95728c20a700436b4f3a8e699a99235748175)